connect/src/com/telenor/connect/ConnectSdk.java

```diff
@@ -67,7 +67,7 @@
     private static ConnectIdService connectIdService;
     private static Context context;
     private static boolean confidentialClient;
-    private static WellKnownAPI.WellKnownConfig wellKnownConfig;
+    private static volatile WellKnownAPI.WellKnownConfig wellKnownConfig;
```

Comment:
is this variable now unused?

Reply:
@vlarsen Variable is still used in other places.